src/frontend/hooks/useDraftObservation.ts

```diff
@@ -68,10 +73,10 @@ export const useDraftObservation = () => {
   }, [cancelPhotoProcessing, clearPersistedDraft]);
 
   const newDraft = useCallback(
-    (capture?: Promise<CapturedPictureMM>) => {
+    (photoPromiseWithMetadata: PhotoPromiseWithMetadata | undefined) => {
```

Comment:
currently causes a TS error [here](https://github.com/digidem/comapeo-mobile/blob/c079a43c24597e6aaef6fc78bf36ae5833d8ff40/src/frontend/screens/MapScreen/index.tsx#L52) because of the subtle signature change here around optional values. Either:

- update the call site to explicitly pass `undefined`
- update this signature to be `photoPromiseWithMetadata?: PhotoPromiseWithMetadata`